facia/app/controllers/FaciaController.scala

```diff
@@ -158,7 +158,7 @@ trait FaciaController extends BaseController with Logging with ImplicitControlle
         ))
         // setting Vary header can be expensive (https://www.fastly.com/blog/best-practices-using-vary-header)
         // only set it for fronts with targeted collections
-        if (TargetedCollections.pageContainsTargetedCollections(faciaPage)) {
```

Comment:
This check didn't work, as by this point `faciaPage` has already had the targeted container removed